### PR TITLE
Add boot script to project

### PR DESCRIPTION
**Pull request by piofinn**
## 📥 Proposed changes

* Add command `yarn boot` to install dependencies and build packages
* Add command `yarn reboot` to clean built files, and then install/build (useful when changing branches)
* Add shortcut command to start portal from root (`yarn dev` or `yarn start`)
* Update `yarn clean` scripts to remove all built files
* Add documentation of new commands

## ☑️ Submission checklist

-   [x] I have read the [CONTRIBUTING](https://github.com/fremtind/jokul/blob/master/CONTRIBUTING.md) doc
-   [x] `yarn build` works locally with my changes
-   [x] I have added tests that prove my fix is effective or that my feature works
-   [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -16,7 +16,10 @@
         "test:unit": "jest -c './jest/jest.unit.js'",
         "prerelease": "yarn build",
         "release": "lerna publish",
-        "start": "lerna run --scope @fremtind/portal dev --stream"
+        "dev": "lerna run --scope @fremtind/portal dev --stream",
+        "start": "yarn run dev",
+        "boot": "yarn install && yarn build",
```
Comment (lfbergee) [inline package.json]:
```suggestion
        "boot": "run-s install build",
```

File: package.json [hunk 2]
```diff
@@ -16,7 +16,10 @@
...
+        "reboot": "yarn run clean && rimraf node_modules/ && yarn run boot"
```
Comment (lfbergee) [inline package.json]:
```suggestion
        "clean:" run-p clean:*,
        "clean:root: "rimraf node_modules/",
        "clean:packages: "lerna run --parallel clean",
        "reboot": "run-s clean boot"
```

File: package.json (hunk 1 above)
Comment (lfbergee) [reply package.json]: Er det ikke ofte litt issues med windows og &&?

Comment (piofinn) [reply package.json]: Sant det...

File: package.json (hunk 2 above)
Comment (piofinn) [reply package.json]: Liten typo i forslaget, men tar med dette!

File: package.json [hunk 3]
```diff
@@ -2,7 +2,9 @@
     "name": "@fremtind/jkl",
     "private": true,
     "scripts": {
-        "clean": "lerna run --parallel clean",
+        "clean": "run-p clean:*",
+        "clean: root": "rimraf node_modules/",
+        "clean: packages": "lerna run --parallel clean",
```
Comment (lfbergee) [inline package.json]:
```suggestion
        "clean:root": "rimraf node_modules/",
        "clean:packages": "lerna run --parallel clean",
```

Comment (lfbergee) [reply package.json]: Veldig fan av suggestion og multiline suggestions og add suggestion to batch!

Comment (piofinn) [reply package.json]: Bra catch! Og ja, det er sykt nice! Jeg glemmer å bruke add to batch, men er sykt smooth å kunne ta inn endringer med ett trykk.

**Pull request merged**